tldw_Server_API/app/core/DB_Management/media_db/repositories/media_repository.py

```
@@ -0,0 +1,864 @@
+            source_hash_norm = source_hash_str if source_hash_str else None
+        raw_url_input = str(url).strip() if url is not None else ""
+        if raw_url_input:
+            dedupe_url_candidates = media_dedupe_url_candidates(raw_url_input)
+            url = normalize_media_dedupe_url(raw_url_input) or raw_url_input
+        else:
+            url = f"local://{media_type}/{content_hash}"
+            dedupe_url_candidates = (url,)
+
+        final_chunk_status = "completed" if chunks is not None else "pending"
+
+        logging.info("add_media_with_keywords: url={}, title={}, client={}", url, title, client_id)
```

Ruff (0.15.6)

[error] Too many arguments for `logging` format string

(PLE1205)